sys/riscv/riscv/exception.S

```diff
@@ -186,24 +195,42 @@ __FBSDID("$FreeBSD$");
 	csrr	t0, sepc
 	sd	t0, (TF_SEPC)(sp)
 #endif
+#ifdef __CHERI_PURE_CAPABILITY__
+	csrr	t0, sstatus
```

Comment:
Maybe this should be using `STORE_LONG` macros like #935 ?

Reply:
I changed these but perhaps we want to cleanup the rest of the assembly with macros as well at some point.